### PR TITLE
Feat administration

### PR DESCRIPTION
**Pull request by remcocgi**
connects to osgp/platform#189 connects to osgp/platform#190

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Shared?pullRequest=35) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

**Comment by bvdzwet** [comment]
DTO value objects should be about some data inside the messages. They normally should not be concerned about things like values being retrieved or set. The device identification does not need to be included in the DTO. Examples of DTOs that better focus on the contents are the AlarmNotifications and the ConfigurationObject.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/SetAdministration.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    private boolean enabled;
+
+    private String deviceIdentification;
+
+    public boolean isEnabled() {
+        return this.enabled;
+    }
+
+    public void setEnabled(final boolean enabled) {
```
Comment (bvdzwet) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/SetAdministration.java]: Value objects should be immutable.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/DeviceFunction.java [hunk 2]
```diff
@@ -43,5 +43,7 @@
     SET_ALARM_NOTIFICATIONS,
     SET_CONFIGURATION_OBJECT,
     SET_ACTIVITY_CALENDAR,
-    REQUEST_ACTUAL_METER_DATA
+    REQUEST_ACTUAL_METER_DATA,
+    GET_ADMINISTRATION,
+    SET_ADMINISTRATION
```
Comment (bvdzwet) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/DeviceFunction.java]: I think the name administration deviates too far from what these functions are about.
A better name would be something like GET_ADMINISTRATIVE_STATE (and the equivalent for SET).
DSMR names the object being get/set "Administrative in/out", but with values "0: Undefined", "1: Meter is administrative off (Opt Out)", "2: Meter is administrative on (Default)" this name appears badly chosen.

**Pull request merged**